python/hail/expr/functions.py

```diff
@@ -872,30 +1254,22 @@ def qpois(p, lamb, lower_tail=True, log_p=False):
     """
     return _func("qpois", TInt32(), p, lamb, lower_tail, log_p)
 
-@typecheck(stop=expr_int32)
-def range(stop):
```

Comment:
Is this being removed on purpose?

Reply:
yes. It was totally nonfunctional, since "range" was redefined by the "range" with the 3-argument signature below (overloading doesn't work in python)